pester_test.go

```diff
@@ -152,7 +152,7 @@ func TestCustomLogHook(t *testing.T) {
 
 	// in the event of an error, let's see what the logs were
 	if expectedRetries != len(errorLines) {
-		t.Errorf("Expected %d lines to be emitted. Got %d", expectedRetries, errorLines)
+		t.Errorf("Expected %d lines to be emitted. Got %d", expectedRetries, len(errorLines))
```

Comment:
`errorLines := []ErrEntry{}`
The current test, with the `%d`, is just saying "hey, these numbers don't match." 
If you want to see the actual individual `ErrEntry` values, we would have to change `%d` to `%v`. By keeping the `%d`, I would expect the fmt statement to say something like:
```
Expected 3 lines to be emitted. Got %!d(ErrEntry=something)
```

Reply:
Without this fix, I get the following failure:

# _/Users/raj/pester
./pester_test.go:155: Errorf format %d has arg errorLines of wrong type []pester.ErrEntry
FAIL	_/Users/raj/pester [build failed]

Reply:
doh! I totally read it backwards (facepalm).